jax/experimental/jax2tf/tests/primitive_harness.py

```diff
@@ -186,6 +186,46 @@ def parameterized(harness_group: Iterable[Harness],
   ]
 )
 
+
+lax_sort = jtu.cases_from_list( # one array, random data, all axes, all dtypes
+  Harness(f"one_array_shape={jtu.format_shape_dtype_string(shape, dtype)}_axis={dimension}_isstable={is_stable}",
+          lax.sort,
+          [RandArg(shape, dtype), StaticArg(dimension), StaticArg(is_stable)],
+          shape=shape,
+          dimension=dimension,
+          dtype=dtype,
+          is_stable=is_stable)
+  for dtype in jtu.dtypes.all
+  for shape in [(5,), (5, 7)]
+  for dimension in range(len(shape))
+  for is_stable in [False, True]
+) + jtu.cases_from_list( # one array, potential edge cases
+  Harness(f"one_array_shape={jtu.format_shape_dtype_string(arr.shape, arr.dtype)}_axis={dimension}_isstable={is_stable}",
```

Comment:
The names of the harnesses have to be unique. This works here because the `arr.shape` is different from what you have used already. I would make it even safer (and easier to find) by changing the `one_array_` prefix to `one_special_array`